doc/classes/String.xml

```diff
@@ -392,12 +392,20 @@
 			<return type="String">
 			</return>
 			<description>
+				Escapes (encodes) a string to URL friendly format. Also referred to as 'URL encode'.
+				[codeblock]
+				print("https://example.org/?escaped=" + "Godot Engine:'docs'".http_escape())
+				[/codeblock]
 			</description>
 		</method>
 		<method name="http_unescape">
 			<return type="String">
 			</return>
 			<description>
+				Unescapes (decodes) a string in URL encoded format. Also referred to as 'URL decode'.
+				[codeblock]
+				print("https://example.org/?escaped=" + "Godot%20Engine%3A%27docs%27".http_unescape() )
```

Comment:
There's still an extraneous space before the closing `)`, otherwise it's good :)